v2/api-validator/README.md

```diff
@@ -1,54 +1,128 @@
-# Fireblocks Network Link v2 API Validator
+<center><img width="400px" src="logo.svg" alt="fireblocks-logo></center>
 
-This project contains the Fireblocks Network Link v2 API validation tool.
-The tool is built to be executed as a stand-alone application that sends various 
-HTTP requests to the system under test to validate the correctness of the API
-implementation.
+<br/>
+<br/>
+<center>
+<h1>Fireblocks Network Link v2 API Validator</h1>
+<ul>
+    <li style="display:inline-block; margin: 0 10px;"> <a href="../openapi/fb-unified-openapi.yaml"> Network Link v2 API Specification </a></li>
+</ul>
+</center>
 
-## Prerequisites
+<br/>
 
-- [nvm](https://github.com/nvm-sh/nvm)
 
+## Intro
 
-## Quick start
+Fireblocks Network Link v2 API Validator is a specialized software tool created to assist third-party providers in ensuring the accuracy and proper functionality of the Application Programming Interface (API) they have developed. 
 
-Setup:
+The primary purpose of this validation tool is to offer a comprehensive range of tests that thoroughly examine the API's implementation. The tool assesses whether the API operates correctly and adheres to the specified standards and requirements. By conducting these tests, partners can identify potential issues, bugs, or inconsistencies in their API. 
 
-```shell
+The Network Link v2 API Validator is designed as a standalone application, meaning it can be run independently of any other software. It functions by initiating HTTP requests to a server that has been configured for testing purposes. These requests are structured to simulate various scenarios and interactions that the API might encounter in real-world usage.
+
+The validation process encompasses a series of tests that cover different aspects of the API's behavior. This could include tests to verify data input and output, proper handling of different HTTP methods, authentication and authorization mechanisms, error handling, and response times. By running these tests, partners can gain confidence that their API functions as intended and meets the expected standards of performance and security.
+
+<br/>
+
+## Prerequisites
+
+- [Node Version Manager (nvm)](https://github.com/nvm-sh/nvm)
+
+<br/>
+
+## Setup
+
+```js
 nvm install 18.14.2
 nvm use
 npm install
 ```
+<br/>
 
+## Running tests
```

Comment:
Isn't it too overwhelming to dive directly into the configuration process? I'd propose first to run the tests with the mock server, be convinced that everything works and only then try to reconfigure. This way, when there are errors in the configuration, and there will be errors, the user will know that something is wrong in their configuration and not that the tool doesn't work properly?

Reply:
Sounds reasonable to me. I'll swap the order.